src/snovault/storage.py

```diff
@@ -48,9 +49,7 @@ def includeme(config):
     _DBSESSION = registry[DBSESSION]
     if registry.settings.get('blob_bucket'):
         registry[BLOBS] = S3BlobStorage(
-            registry.settings['blob_bucket'],
-            read_profile_name=registry.settings.get('blob_read_profile_name'),
```

Comment:
Summarizing our conversation about the ini/cfg/s3 aws profile vars and usage.

We should reconsider removing the 'blob_read_profile_name' and 'blob_store_profile_name'.  Or clean up buildout config/ini files.  The current PR seems to work because boto just uses the default profile.  Removing them here leaves unused code in the ini/cfgs and limits our ability to set the profile.  

We could do nothing, which would limit our profile functionality and leave some technical debt in the ini/configuration.   Or we could keep the profile functionality by updating the PR.   Cleaning up the configuration would be a big task.  It would probably require discussion about what functionality we want/need and who should have permission on demo/production.

Reply:
Probably should keep profile_names in case someone else is using them (even though it doesn't seem to make a difference for us). Will update PR.